src/PrestaShopBundle/Resources/config/services/bundle/twig.yml

```diff
@@ -79,3 +79,13 @@ services:
         translator: "@translator"
       tags:
         - { name: twig.extension }
+
+    prestashop.twig.extension.path_with_back_url_extension:
+      class: PrestaShopBundle\Twig\Extension\PathWithBackUrlExtension
+      arguments:
+        - '@twig.extension.routing'
+        - '@prestashop.core.uti.back_url_provider'
+        - '@=service("request_stack") ? service("request_stack") : null'
```

Comment:
I did not know you could do that 🤔

Reply:
Yes, that's really handy if you have "optional" dependencies, like from different libraries/bundles